src/almanack/checks/checks.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
This is probably an annoying comment, but I'm wondering why you chose the word "checks". I know this is semantics, but I'm wondering if something like "metrics", or "repo_health_checks", or "metric_id_dictionary". Right now "checks" feels a bit distant and I don't really know what to expect as contents.

This might also be alleviated by including a brief `README` in the `src/almanack/checks` folder

Reply:
Thanks, I think this is an apt comment. I'll move to `metrics` throughout in replacement for `checks` which is more accurate to what's performed.